content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,145 @@
+.. tip::
+   A user must have :guilabel:`Access Rights` under :guilabel:`Administration` access rights in
+   order to make changes on another user's access rights settings. To access this setting navigate
+   to :menuselection:`Settings app --> Manage users --> User --> Access Rights tab -->
+   Administration`. Once at the setting an already existing administrator must change the setting
+   under :guilabel:`Administration` to :guilabel:`Access Rights`. :guilabel:`Save` the changes to
+   implement the user as an administrator.
 
-.. image:: access_rights/groups-users.png
-   :align: center
-   :alt: View of a group’s form emphasizing the tab users in Odoo
+Users
+=====
+
+The :ref:`Access Rights <users/add-individual>` for individual users are set when the user is added
+to the database, but they can be adjusted at any point in the user's profile.
 
-*Inherited* means that users added to this application group are automatically added to the
-following ones. In the example below, users who have access to the group *Administrator* of *Sales*
-also have access to *Website/Restricted Editor* and *Sales/User: All Documents*.
+To make changes to a user's rights, first activate Odoo's :ref:`developer mode <developer-mode>`,
+then go to :menuselection:`Settings --> Users & Companies --> Users`. Click on the user to edit.
 
-.. image:: access_rights/groups-inherited.png
+.. image:: access_rights/navigate-to-users-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab inherited in Odoo
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Users.
 
-.. important::
-   Remember to always test the settings being changed in order to ensure that they are being applied
-   to the needed and right users.
+At the user's profile page, click on the :guilabel:`Access Rights` tab and scroll down to view the
+current permissions. For each app, use the drop-down menu to select what level of permission this
+user should have. The options vary for each section yet the most common are: *Blank/None*, *User:
+Own Documents*, *User: All Documents*, or *Administrator*.
 
-The *Menus* tab is where you define which menus (models) the user can have access to.
+Administration includes either :guilabel:`Settings` or :guilabel:`Access Rights`.
 
-.. image:: access_rights/groups-menus.png
+.. image:: access_rights/user-permissions-dropdown-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab menus in Odoo
+   :alt: Open the drop-down menu next to each app to set the user's level of permissions.
+
+Creating and modifying groups
+=============================
 
-*Access Rights* rules are the first level of rights. The field is composed of the object name, which
-is the technical name given to a model. For each model, enable the following options as appropriate:
+Groups are app-specific sets of permissions that are used to manage common access rights for a large
```

Comment:
```suggestion
*Groups* are app-specific sets of permissions that are used to manage common access rights for a large
```
(Totally optional, but since you are defining the term, I thought it made sense to italicize the first usage)